auth/login.html

```diff
@@ -9,18 +9,17 @@
 </head>
 <body class="body">
     <div>
-        <form id="registrationForm" class="forms">
+        <form id="registrationForm" class="forms" action="login_handler.php" method="post">
             <div>
-                <div>
-                    <label for="fname" class="labels">First name:</label><br>
-                    <input type="text" id="nameLogin" name="fname" class="inputs"><br>
-                </div>
+                <div class="gap">
+                    <label for="nemail" class="labels">Email:</label><br>
+                    <input type="email" id="emails" name="nemail" class="inputs"><br><br>
+                </div class="gap">
```

Comment:
```suggestion
                </div>
```
why provided class on closing tag?